<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpPopup</class>
 <widget class="QWidget" name="HelpPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help Window</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>10</number>
  </property>
  <widget class="QTabWidget" name="createTab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>541</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>Create</string>
    </attribute>
    <widget class="QTextBrowser" name="createText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Toolbar</string>
    </attribute>
    <widget class="QTextBrowser" name="toolbarText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Animation Preview</string>
    </attribute>
    <widget class="QTextBrowser" name="animationPreviewText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Frame Manager</string>
    </attribute>
    <widget class="QTextBrowser" name="frameManagerText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
